docs/aws-sdk-for-python-with-minio.md

```diff
@@ -0,0 +1,58 @@
+# How to use AWS SDK for Python with Minio Server [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+`aws-sdk-php` is the official AWS SDK for the PHP programming language. In this recipe we will learn how to use `aws-sdk-php` with Minio server.
+
+
+## 1. Prerequisites
+
+Install Minio Server from [here](http://docs.minio.io/docs/minio).
+ 
+## 2. Installation
+
+Install `aws-sdk-python` from AWS SDK for Python official docs [here](https://aws.amazon.com/sdk-for-python/) 
+
+
+## 3. Example
+
+Please replace ``endpoint_url``,``credentials``, ``Bucket`` and ``Object`` with your local setup in this ``example.py`` file.
```

Comment:
'credentials' is not a field here.. 'aws_access_key'_id , aws_secret_access_key'